File: src/models/authenticateResponse.ts
export class AuthenticateResponse {
    accessToken: string;
    refreshToken: string;
    firstName: string;
    lastName: string;

    /**
     *
     */
    constructor(accessToken: string, refreshToken: string, firstName: string, lastName: string) {
        this.accessToken = accessToken;
        this.refreshToken = refreshToken;
        this.firstName = firstName;
        this.lastName = lastName;
    }
}

File: server/DataService/Models/Responses/AuthenticateResponse.cs
using DataService.Models.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataService.Models.Responses
{
    public class AuthenticateResponse
    {
        public string AccessToken { get; set; }
        public string RefreshToken { get; set; }
        public User User { get; set; }
    }
}
